packages/@dcl/ecs/src/initialization.ts

```diff
@@ -37,3 +37,6 @@ if (dcl) {
       .finally(() => engine.update(dt))
   })
 }
+
+export const log = dcl.log
+export const error = dcl.error
```

Comment:
If we expose this methods maybe we should remove the `dcl` types for the scene ?

Reply:
yep, that's the idea :D